src/dev/run_find_plugins_with_circular_deps.ts

```diff
@@ -31,13 +31,9 @@ interface Options {
 type CircularDepList = Set<string>;
 
 const allowedList: CircularDepList = new Set([
+  'src/plugins/charts -> src/plugins/discover',
```

Comment:
Removing the app services-owned circular deps revealed a new circular dep between app-owned plugins, so I added it to the `allowList`.